ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -458,6 +462,51 @@ tests:
     env:
       PROJECT_NAME: machine-config-operator
     workflow: openshift-ci-security
+- as: e2e-aws-ovn-upgrade-ocb
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|OWNERS_ALIASES|LICENSE)$
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+    test:
+    - as: get-prelayering-rpmostree-status
+      commands: |-
+        #!/usr/bin/env bash
+        set -xeuo pipefail
+        run-on-all-nodes --keep-going --exit-zero 'rpm-ostree status'
+      from: zacks-toolbox
+      resources:
+        requests:
+          cpu: 100m
+    - as: enable-layering
+      commands: |-
+        #!/usr/bin/env bash
+        set -xeuo pipefail
+        onclustertesting setup ci --inject-yum-repos --containerfile-path <(curl -Lo https://raw.githubusercontent.com/openshift/machine-config-operator/refs/heads/master/test/e2e-techpreview/Containerfile.cowsay)
+      from: zacks-toolbox
+      resources:
+        requests:
+          cpu: 100m
+    - as: get-postlayering-rpmostree-status
+      commands: |-
+        #!/usr/bin/env bash
+        set -xeuo pipefail
+        run-on-all-nodes --keep-going --exit-zero 'rpm-ostree status'
+      from: zacks-toolbox
+      resources:
+        requests:
+          cpu: 100m
+    - ref: openshift-e2e-test
+    - as: get-posttest-rpmostree-status
+      commands: |-
+        #!/usr/bin/env bash
+        set -xeuo pipefail
+        run-on-all-nodes --keep-going --exit-zero 'rpm-ostree status'
+      from: zacks-toolbox
+      resources:
+        requests:
+          cpu: 100m
+    workflow: openshift-upgrade-aws
```

Comment:
thought: Maybe add another job just like this except set `workflow: openshift-e2e-aws`? This is the typical OpenShift conformance suite running against a cluster with OCL enabled.